CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+<!--
+SPDX-FileCopyrightText: 2019 The curiu authors
+
+SPDX-License-Identifier: LicenseRef-ISC-curiu
+-->
+
+# Contributing
+
+- Add yourself to the [AUTHORS.md](AUTHORS.md) file.
+- Autoformat new/changed code with [`black`](https://github.com/psf/black).
```

Comment:
I have no strong opinion on which tool to use here. `black` seems to be what the cool kids use.
I just want that we use one such tool -- I don't want to waste much time on discussing formatting conventions ;).

Reply:
I personally love to use black, it gives me the freedom to totally forget about the style! :laughing: